week-3/implement/rotate-char.js

```diff
@@ -8,6 +8,21 @@
 
 // Acceptance criteria:
 
+function rotateCharacter(char, shift) {
+  const alphabet = "abcdefghijklmnopqrstuvwxyz";
+  const alphabetCaps = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+  if ((alphabet.includes(char) && shift > 0) || shift < 0) {
```

Comment:
For some of your logs, the function is returning the same character back. This may be due to a mistake in the our instructions. Why do you need the first check in your function?